pontoon/base/tests/test_models.py

```diff
@@ -544,14 +544,14 @@ def test_period_filters(self):
 
         top_contributors = User.translators.with_translation_counts(aware_datetime(2015, 6, 10))
 
-        assert_equal(top_contributors.count(), 1)
+        assert_equal(len(top_contributors), 1)
```

Comment:
I don't think this is better in terms of perf. The only use you make of ``top_contributors``' values is of the 1st element, so by using ``len()`` here, you force it to select all results, when afterwards you only use the first one. ``count()`` seems better here.

But, the key point is that you're trying to optimize unit tests, which is not a high priority at the moment, so you could probably avoid this altogether.

Reply:
`top_contributors` is no longer a QuerySet, it's a list.

Reply:
Aha, I missed that, my bad. :)